thinc/layers/hashembed.py

```diff
@@ -9,7 +9,7 @@
 from ..util import partial
 
 
-InT = Union[Ints2d, Ints1d]
+InT = Ints1d
```

Comment:
Here the declared and documented type included Ints2d, but the backprop method appears not to support it.